app/policies/batch_policy.rb

```diff
@@ -13,12 +13,12 @@ def create?
     dia? || same_council?
   end
 
-  def update?
-    (dia? || same_council?) && !record.completed
+  def edit?
```

Comment:
shouldn't the #edit method just call the `update?`?
because i don't think there's any scenario where someone should be able to see and edit for and not be able to called update.

Reply:
over in `ApplicationPolicy` the `update?` calls `edit?` for you.

Reply:
@Br3nda Thanks, I made this change.